apps/root/src/hooks/useTransactionsHistory.tsx

```diff
@@ -73,7 +81,16 @@ function useTransactionsHistory(): {
       setParsedEvents([]);
     }
     // Whenever the events, the token list or any pending transaction changes, we want to retrigger this
-  }, [historyEvents, indexing, isLoadingTokenLists, isLoading, tokenList, nonIndexedTransactions, storedWallets]);
+  }, [
+    historyEvents,
+    indexing,
+    isLoadingTokenLists,
+    isLoading,
+    tokenList,
+    nonIndexedTransactions,
+    storedWallets,
+    nativePrices,
```

Comment:
This doesnt impact performance right? You can easily check it with the activity and portfolio tables checking if they  keep rendering correctly or if there is a log between scrolling and the rows rendering